src/chapters/classes-part2/equals-shortcut.rst

```diff
@@ -0,0 +1,72 @@
+Visual Studio Generator Shortcut
+================================
+
+Instead of cutting, pasting, and refactoring old code to ensure that you create a well-structured ``GetHashCode()`` method whenever you define your own ``Equals()`` method, you can use Visual Studio’s code generation tool!
+Just right-click in your class file on the class name and follow these prompts.
+
+Let’s use a Course class to demonstrate:
```

Comment:
just want to make sure you've check to see if the action is the same for windows visual studio?